playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -185,8 +185,8 @@
       register: shipyard_desc_action
       failed_when: "'failed' in shipyard_desc_action.stdout"
       until: shipyard_desc_action.stdout.find('Processing') < 0 and shipyard_desc_action.stdout.find('running') < 0
-      retries: "{{ airship_deploy_openstack_timeout | int * 2 }}"
-      delay: 30
+      retries: "{{ airship_deploy_openstack_timeout | int }}"
```

Comment:
this effectively sets the same timeout (half retries, double delays). This proves insufficient at least on our internal deployments (times out). I would let the number of retries as they were but with the increased delays.

Reply:
Yes, my point here was not changing he overall timeouts, only the cleanup so that there's not that many retries for actions that take too much time